styles.css

```diff
@@ -127,3 +132,165 @@ body {
   opacity: 1;
   width: 100%;
 }
+
+@media (min-width: 1900px) {
+
+  .wrapper {
+    width: 85%;
+  }
+
+  .logo {
+    padding: 40px 0 40px 97px;
+  }
+
+  .logo__circle_color_blue, .logo__circle_color_dark-blue {
+    margin-left: 53px;
+  }
+
+  .menu {
+    padding-top: 0;
+    padding-right: 39px;
+  }
+
+  .menu__item {
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+  }
+
+  .header {
+    margin-bottom: 10%;
+  }
+
+  .main {
+    padding: 38px 50% 0 50px;
+    background: top -30px right 6% /47% no-repeat url(img/img.svg);
+
+  }
+
+  .main__title {
+    font: bold 66px/88px 'pt_serif', arial, helvetica, sans-serif;
+    margin-bottom: 35px;
+  }
+
+  .main__description {
+    max-width: 60%;
+    font: 22px/26px Roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    font: 500 18px/21px Roboto, arial, helvetica, sans-serif;
+    padding: 17px 28px;
+    margin-left: 2px;
+    margin-bottom: 25%;
+  }
+}
+
+@media (max-width: 768px) {
+  #checkbox {
+    margin: 5% 0 0 65%;
+    opacity: 1;
+  }
+
+  #checkbox:checked~.burger::before {
+    transform: rotate(45deg);
+  }
+
+  #checkbox:checked~.burger::after {
+    transform: rotate(135deg);
+    margin-top: -10px;
+  }
+
+  #checkbox:checked~.burger__menu {
+    border: none;
+  }
+
+  #checkbox:checked~.menu {
+    display: block;
+    text-align: center;
+  }
+
+  #checkbox:checked~.menu__item {
+    display: inline-block;
+    font: 500 30px/35px Roboto, arial, helvetica, sans-serif;
+    transition: .5s;
+    margin-left: -50%;
+  }
+
+  .wrapper {
+    width: 100%;
+    margin: auto;
+  }
+
+  .logo {
+    padding-top: 35px;
+  }
+
+  .logo__text {
+    display: none;
+  }
+
+  .logo__circle_color_dark-blue {
+    margin-left: 23px;
+  }
+
+  .logo__circle_color_blue {
+    margin-left: 23px;
+  }
+
+
+  .burger {
+    display: block;
+    margin: 27px 8px 0 0;
+  }
+
+  .burger__menu {
+    display: flex;
+    width: 25px;
+    padding-top: 4px;
+    border-bottom: 3px solid #0056ec;
+  }
+
+  .burger::after, .burger::before {
+    display: flex;
+    position: relative;
+    content: "";
+    border-bottom: 3px solid #0056ec;
+  }
+
+  .burger::after, .burger::before {
+    padding-bottom: 4px;
+  }
+
+  .header {
+    margin-bottom: auto;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .main {
+    text-align: center;
+    padding: 70% 0 0 0;
+    background-size: 80%;
+    background-position: top 5px right 26px;
+  }
+
+  .main__title {
+    font: bold 30px/40px 'pt_serif', arial, helvetica, sans-serif;
+    padding-top: 17px;
+    margin-left: 14px;
+    margin-bottom: 32px;
+  }
+
+  .main__description {
+    margin-bottom: 25px;
+    max-width: 68%;
+    font: 16px/19px Roboto, arial, helvetica, sans-serif;
+    margin-left: 19%;
+  }
+
+  .main__button {
+    font: 500 14px/16px Roboto, arial, helvetica, sans-serif;
+    margin-left: 5%;
+    }
```

Comment:
Codacy found an issue: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/FirstWebsait/pullRequest?prid=6278784)